runtime/vm/jvminit.c

```diff
@@ -663,6 +663,11 @@ freeJavaVM(J9JavaVM * vm)
 		vm->cifArgumentTypesCache = NULL;
 	}
 
+	/* Delete the layout string hashtable if exists. */
+	if (NULL != vm->layoutStrFFITypeTable) {
+		releaseLayouStrFFITypeTable(vm);
+	}
```

Comment:
Suggest the argument just be the hash table, and the field be cleared here. The duplicate check that the pointer is not null can be removed from `releaseLayoutStrFFITypeTable()`.

Reply:
Fixed as suggested above.